data/styles/outdoors/include/priorities_4_overlays.prio.txt

```diff
@@ -1212,13 +1212,13 @@ leisure-track-area                                  # caption z16- (also has are
 power-generator                                     # icon z17-
 power-generator-gas                                 # icon z17-
 power-generator-hydro                               # icon z17-
-power-generator-wind                                # icon z17-
+power-generator-wind                                # icon z13- (also has caption(optional) z16-)
 power-plant                                         # icon z17-
 power-plant-coal                                    # icon z17-
 power-plant-gas                                     # icon z17-
 power-plant-hydro                                   # icon z17-
 power-plant-solar                                   # icon z17-
-power-plant-wind                                    # icon z17-
+power-plant-wind                                    # icon z14- (also has caption(optional) z16-)
```

Comment:
power plant shouldn't be shown later (z14) than individual generators (z13), right?
make it z13- also please
and higher priority